README.md

```diff
@@ -16,9 +16,36 @@ Architecture decision records can be found in the
 
 ## Setting up the app in development
 
-1. Run `bundle install` to install the gem dependencies
-2. Run `bundle exec rails db:setup` to set up the database development
-3. Run `bundle exec rails server` to launch the app on http://localhost:3000
+1. In order to integrate with DfE Sign-in's Open ID Connect service we are
+   required to communicate over https in development. Create a self-signed
+   development certificate.
+   - Run `openssl req -x509 -sha256 -nodes -newkey rsa:2048 -days 365 -keyout localhost.key -out localhost.crt`
+   - Open Keychain Access on your Mac and go to the Certificates category in
+     your System keychain. Once there, import the `localhost.crt`
+     `File > Import Items`. Double click the imported certificate and change the
+     “When using this certificate:” dropdown to `Always Trust` in the Trust
+     section.
+2. Run `bundle install` to install the gem dependencies
+3. Run `bundle exec rails db:setup` to set up the database development
+4. Run `foreman start` to launch the app on https://localhost:3000
```

Comment:
Any chance we could binstub `foreman`?

Reply:
I've suggested we add it to the `Gemfile` `development` group so we can run it with bundler instead.

Reply:
I've updated the readme to say `bundle exec foreman start`